Application.hs

```diff
@@ -79,7 +80,14 @@ makeFoundation appSettings = do
 -- applying some additional middlewares.
 makeApplication :: App -> IO Application
 makeApplication foundation = do
-    logWare <- mkRequestLogger def
+    logWare <- makeLogWare foundation
+    -- Create the WAI application and apply middlewares
+    appPlain <- toWaiAppPlain foundation
+    return $ logWare $ defaultMiddlewaresNoLogging appPlain
+
+makeLogWare :: App -> IO Application
```

Comment:
I think this is supposed to return `IO Middleware`. Note: Middleware isn't currently being imported into this file, and `wai` isn't in the `build-depends` in the cabal file.